proto/viam/app/model/v1/model.proto

```diff
@@ -48,6 +56,21 @@ message DeployRequest {
   DeployMetadata metadata = 1;
 }
 
+message Model {
+  string model_name = 1;
```

Comment:
Think can rename `model_name` and `model_size` to `name` and `size_bytes`

supernit: perhaps switch fields 2 and 3 to keep the above two fields as consecutive.

Reply:
changed!